massa-api-exports/Cargo.toml

```diff
@@ -8,7 +8,7 @@ edition = "2021"
 paginate = "1.1.11"
 displaydoc = "0.2"
 thiserror = "1.0"
-jsonrpsee = { git = "https://github.com/paritytech/jsonrpsee.git", rev = "118acc3", features = ["jsonrpsee-core", "jsonrpsee-types"] }
+jsonrpsee = { git = "https://github.com/paritytech/jsonrpsee.git", rev = "ab097c8", features = ["jsonrpsee-core", "jsonrpsee-types"] }
```

Comment:
Why don't you just use a tagged/published version like other dependencies ?

Reply:
To get the latest bug fixes, but It seems to be a runtime issue. I'll check this quickly and be back

Reply:
In order to the latest bugfixes

Reply:
They didn't add much things since the last version 0.18.2 its doesn't seems that there is bugs that impact us in this version. It's better to not rely on a commit but on a tagged version in this case I think.

Reply:
done